src/java/io/bazel/rulesscala/scalac/ScalacWorker.java

```diff
@@ -255,25 +255,35 @@ private static void compileScalaSources(CompileOptions ops, String[] scalaSource
       throw new RuntimeException("Unable to write statsfile to " + ops.statsfile, ex);
     }
 
-    CompositeReporter compositeReporter = (CompositeReporter) reporterField.get(comp);
```

Comment:
which older versions does this try to support? Which usecase exactly?

Reply:
When running the `test_version.sh` for scala version `2.11.12`, a protoreporter was received not a composite one

Reply:
Where was it recurved? I think I'm missing a bit here.
You expect either a composite (with proto/console inside) or a console reporter.
You never expect a proto reporter directly

Reply:
I believe this problem is longer pertainable due to the most recent changes